### PR TITLE
Make doc structure consistent and up-to-date   

### PR DESCRIPTION
**Pull request by Julian-O**
This is part of an effort to make the Kivy sibling projects' documentation
structure consistent and up-to-date.

Unrelated changes:
* Readme Copy-edit. Reformat markdown table
* RST added structure, including API page
* Congrats, promoted from "beta" to "stable".
* Fixed some (but not all) unrelated PEP8 changes

CHECKLIST

* CONTRIBUTING.md
   [x] If repo takes user contributions, is present
   [x] In root dir (not .github dir)
   [x] Refers to kivy/kivy Contribution Guidelines.
   
* LICENSE
   [x] If repo takes user contributions, is present.
   [x] Acknowledges the range of years to 2023.
   [x] Acknowledges Kivy Team and other contributors
   [x] Mentions it is an MIT License.
   
* CODE_OF_CONDUCT.md
   [x] If repo takes user contributions or hosts discussions, is present.
   [x] Refers to kivy/kivy CODE_OF_CONDUCT.md
   
* CONTACT.md
   [x] Refers to kivy/kivy CONTACT.md

* FAQ.md
   [NA] If repo is big enough for RST documentation, is present.
   (RST documentation exists but is trivial)
   
* README:
   [x] Is a Markdown file.
   [x] Describes the project.
   [x] Describes its place in the Kivy sibling projects.
   [x] If CONTRIBUTING exists, mentions it.
   [x] If LICENSE exists, mentions it.
   [x] If CODE_OF_CONDUCT exists, mentions it.
   [x] Mentions kivy/kivy CONTACT.md
   [NA] Uses Python syntax colouring for embedded Python code.
   [x] Uses badges to display current status.

* RST documentation, if present
   [x] Describes the project.
   [x] Describes its place in the Kivy sibling projects.
   [x] Mentions LICENSE.
   [x] Mentions CONTRIBUTING
   [NA] Mentions FAQ
   [x] conf.py mentioned Kivy Team and other contributors
		- copyright, latex_documents, man_pages, texinfo documents
   
* WORKFLOWS
   [x] NO_RESPONSE.yml is present if the repo has awaiting_reply tag.
   [x] NO_RESPONSE uses latest script versions.
   [x] SUPPORT.yml is present if the repo has a `support` tag.
   [x] SUPPORT.yml refers to kivy/kivy CONTACT.md

* setup.py/cfg, if present and on PyPI
   [x] Supplies description to PyPI
   [x] Supplies Python versions to PyPI
   [x] Supplies Documentation, if any, to PyPI

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
-   sphinx-quickstart on Wed Jul  3 15:18:02 2013.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 Welcome to Plyer
 ================
 
-Plyer is a Python library for accessing features of your hardware / platforms.
+Plyer is a platform-independent Python API for accessing features
+of various hardware platforms, especially Android and iOS devices.
```
Comment (misl6) [inline docs/source/index.rst:5]:
```suggestion
Plyer is a platform-independent Python API for accessing features
of various platforms (Android, iOS, macOS, Linux and Windows).
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+
+Plyer is managed by the `Kivy Team <https://kivy.org/about.html>`
+and is suitable for use with Kivy apps.
+
+Plyer is released and distributed under the terms of the MIT license. Our
+`latest license <https://github.com/kivy/kivy/blob/master/LICENSE>`_
```
Comment (misl6) [inline docs/source/index.rst:11]:
```suggestion
`latest license <https://github.com/kivy/plyer/blob/master/LICENSE>`_
```

File: setup.py [hunk 3]
```diff
@@ -56,7 +56,9 @@
 setup(
     name='plyer',
     version=plyer.__version__,
-    description='Platform-independent wrapper for platform-dependent APIs',
+    description='A platform-independent Python API for accessing features of '
+                'various hardware platforms, especially Android and iOS '
```
Comment (misl6) [inline setup.py:60]: Maybe the same description (for SEO purposes) that I have suggested in README?

**Review by misl6 — CHANGES_REQUESTED** [review]
Hi @Julian-O !

I've left some comments, let me know what you think about it.

File: README.md [hunk 4]
```diff
@@ -1,134 +1,170 @@
 # Plyer
 
-Plyer is a platform-independent api to use features commonly found on various
-platforms, notably mobile ones, in Python.
+Plyer is a platform-independent Python API for accessing features
+of various hardware platforms, especially Android and iOS devices.
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md:4]: Okay, but I have reworded to keep in the hardware part, because that is the element that unifies these APIs.

**Pull request merged**